tests/ui/solver-attribute/no-arg/expected

```diff
@@ -0,0 +1,6 @@
+error: The `#[kani::solver]` attribute expects a single argument. Got 0 arguments.
+test.rs:5:1
+  |
+5 | #[kani::solver]
```

Comment:
nit: remove the line number from the expected files and add "\" to the end of each line.

Reply:
Done.